.idea/misc.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this .idea stuff. You might want to add it to your `.git/info/exclude` or even better something like `~/.gitignore_global`